Copying.ipynb

```diff
@@ -0,0 +1,1063 @@
+{
```

Comment:
<p>Type: it should be  mutable, also better use find and replace, it happend on several occasions :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuefische/ds-meetups/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>thanks :)</p>